app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -10,6 +10,14 @@ HTMLDocument.prototype.ready = new Promise((resolve) => {
 
 // If localStorage.getItem('shouldRedirectToOnboarding') is not set, i.e. null, that means we should redirect.
 function redirectableLocation() {
+  return (
+    window.location.pathname !== '/onboarding' &&
+    window.location.pathname !== '/signout_confirm' &&
+    window.location.pathname !== '/privacy'
+  );
```

Comment:
Refactor (blocking): Use [Array.prototype.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes).

```suggestion
  return !['/onboarding', '/signout_confirm', '/privacy'].includes(
    window.location.pathname,
  );
```

Reply:
TIL! Updated in both functions, @nickytonline!